src/vector/nalgebra_serial.rs

```diff
@@ -33,6 +35,42 @@ impl<'a, T: Scalar> VectorView<'a> for DVectorView<'a, T> {
     fn into_owned(self) -> Self::Owned {
         self.into_owned()
     }
+    fn scalar_mul(&self, rhs: Self::T) -> Self::Owned {
+        self * rhs
+    }
+}
+impl<'a, T: Scalar> Mul<Scale<T>> for DVectorView<'a, T> {
+    type Output = DVector<T>;
+    fn mul(self, rhs: Scale<T>) -> Self::Output {
+        self * rhs.value()
+    }
+}
+impl<'a, T: Scalar> Mul<Scale<T>> for DVectorViewMut<'a, T> {
+    type Output = DVector<T>;
+    fn mul(self, rhs: Scale<T>) -> Self::Output {
+        self * rhs.value()
+    }
+}
+impl<T: Scalar> Mul<Scale<T>> for DVector<T> {
+    type Output = DVector<T>;
+    fn mul(self, rhs: Scale<T>) -> Self::Output {
+        self * rhs.value()
+    }
+}
+
+impl<T: Scalar> MulAssign<Scale<T>> for DVector<T> {
+    fn mul_assign(&mut self, rhs: Scale<T>) {
+        *self = &*self * rhs.value();
+    }
+}
+impl<'a, T: Scalar> MulAssign<Scale<T>> for DVectorViewMut<'a, T> {
```

Comment:
```suggestion
impl<'a, T: Scalar> MulAssign<Scale<T>> for DVectorViewMut<'a, T> 
where
   for <'b> &'b DVectorViewMut: VectorRef<'b>
{
```
This might allow you to multiply a &DVector but a scalar (you might need to get rid of the mut first)

Reply:
Found a different way to fix it